deno/payloads/v8/invite.ts

```diff
@@ -3,12 +3,23 @@
  */
 
 import type { APIPartialChannel } from './channel.ts';
-import type { APIPartialGuild } from './guild.ts';
+import type { APIGuild } from './guild.ts';
 import type { APIApplication } from './application.ts';
 import type { APIInviteStageInstance } from './stageInstance.ts';
 import type { APIUser } from './user.ts';
 import type { APIGuildScheduledEvent } from './guildScheduledEvent.ts';
 
+export interface APIInviteGuild
+	extends Pick<
+		APIGuild,
+		'id' | 'name' | 'description' | 'features' | 'verification_level' | 'nsfw_level' | 'premium_subscription_count'
+	> {
+	splash: null;
```

Comment:
Are you sure these are null? Can you fetch, say, djs's invite and see what it returns? 👀

Reply:
`splash`, `banner`, `icon` and `vanity_url_code` gets returned

Reply:
thanks, I'll add those back